packages/generators/src/init-generator.ts

```diff
@@ -19,20 +19,6 @@ export default class InitGenerator<
   }
 
   public async prompting(): Promise<void> {
-    if (!existsSync(this.generationPath)) {
-      this.cli.logger.log(
-        `${this.cli.colors.blue(
-          "ℹ INFO ",
-        )} supplied generation path doesn't exist, required folders will be created.`,
-      );
-      try {
-        mkdirSync(this.generationPath, { recursive: true });
-      } catch (error) {
-        this.cli.logger.error(`Failed to create directory.\n ${error}`);
-        process.exit(2);
-      }
-    }
```

Comment:
This is now handled by the yeoman-generator. I've updated the test cases accordingly.